src/components/Article-list.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+
+export default function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetch("https://news-api-project-ek66.onrender.com/api/articles")
```

Comment:
Whilst this works fine, it would be cleaner long term to extract api calling functions into their own utils file.
Also i would recommend using axios rather than fetch when fetching with React